src/Uno.UWP/Devices/Sensors/SimpleOrientationSensorOrientationChangedEventArgs.cs

```diff
@@ -3,7 +3,7 @@
 namespace Windows.Devices.Sensors
 {
 	/// <summary>
-	/// Provides data for the sensor reading�changed event.
```

Comment:
CI was failing with CS1504 for this file (and `BrightnessOverride`). It looks like this is a [bug that was fixed but not yet released](https://developercommunity.visualstudio.com/t/New-version-of-Visual-Studio-for-Mac-17/10114754) (at the time of writing this comment). It's related to file encodings, etc.

So fixing the encoding would avoid seeing the issue.